hardware_accelerators/das-about-dynamic-accelerator-slicer-operator.adoc

```diff
@@ -0,0 +1,92 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="das-about-dynamic-accelerator-slicer-operator"]
+= Dynamic Accelerator Slicer (DAS) Operator 
+include::_attributes/common-attributes.adoc[]
+:context: das-about-dynamic-accelerator-slicer-operator
+
+toc::[]
+
+[IMPORTANT]
+====
+Dynamic Accelerator Slicer Operator (DAS) is a Technology Preview feature and is subject to Technology Preview support limitations. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. 
+
+For more information, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. 
+====
+
+Running data intensive workloads on the platform requires efficient GPU utilization to offset the high cost of the resources involved. One approach for maximizing efficiency is through the use of oversubscription of the GPU resources by time-slicing multiple workloads. This method provides for sharing GPU resources by alternating the execution times of the workloads.
```

Comment:
This is incorrect, DAS operator (formally instaslice) never used time slicing.

Reply:
@harche This doesn't imply that DAS uses time slicing. Rather, it refers to the use of static slicing, as the following paragraph explains and the paragraph after that talks about the advantages of dynamic slicing over time slicing. I only used those paragraphs on static slicing for comparison. 
I can see where this could be confusing. If you don't think this comparison is necessary, I'll remove lines 16-18. Let me know.